cmds/coredhcp/config.yml.example

```
@@ -156,3 +156,9 @@ server4:
         # * lease duration can be given in any format understood by go's
         # "ParseDuration": https://golang.org/pkg/time/#ParseDuration
         - range: leases.txt 10.10.10.100 10.10.10.200 60s
+
+        # optional static routes could be defined as well
```

nit: this should follow the same style as other plugin descriptions:
```suggestion
        # staticroute advertises additional routes the client should install in its routing table
```